core/src/epicli/data/azure/validation/infrastructure/availability-set.yml

```diff
@@ -0,0 +1 @@
+$ref: '#/definitions/unvalidated_specification'
```

Comment:
newline

Reply:
do you mean that we need a newline? If so then do we need it? Most of other file in same dir does not have new line anyway.

IMHO yaml doe snot specify we need it, it is a practice to add `...` for closing or `---` in opening but really need for stream communication of yaml data anyway.